Samples/StreamRecorder/StreamRecorderApp/RMCameraReader.cpp

```diff
@@ -155,13 +155,15 @@ void RMCameraReader::DumpCalibration()
     
     fileExtrinsics.close();
 
-    // Get intrinsics (as LUT)
-    // For this implementation, we save LUT only for the depth camera
-    // This can be changed if one wants to get intrinsics information about VLCs
-    if ((m_pRMSensor->GetSensorType() != DEPTH_LONG_THROW) && (m_pRMSensor->GetSensorType() != DEPTH_AHAT))
-    {
-        return;
-    }
+	// Get intrinsics (as LUT) for all research mode sensors except IMU
+	if (
+		(m_pRMSensor->GetSensorType() == IMU_ACCEL)
+		|| (m_pRMSensor->GetSensorType() == IMU_GYRO)
+		|| (m_pRMSensor->GetSensorType() == IMU_MAG)
+		)
```

Comment:
Currently the StreamRecorder does not even initialize IMU, so this check might not be required.

Reply:
Ok I fixed it thx.